### PR TITLE
docs(rfc): add original rfc

### LINKED ISSUE #30: Add documentation / Design overview
The [originating issue](https://github.com/projectcaluma/caluma/issues/1105) has a nice write-up on what Alexandria is supposed to be. I suggest copying the issue more or less 1:1 into a "docs" folder here and link it from the README

### PR DESCRIPTION
**Pull request by open-dynaMIX**
Closes #30

### CODE REVIEW

**Comment by open-dynaMIX** [comment]
@winged Warm-up PR ;)

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 Document management service
```
Comment (winged) [inline README.md:7]: I'd add a sentence or two to be more specific - "document management service" is rather generic. Samba for instance is a document management service :)

File: docs/original_alexandria_rfc.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+## Rationale
+Documents can currently only be added to a form via file question. These do not provide search, tags, thumbnails and some other features needed for a document management service. File questions can only be used inside a form.
+
+## Goal
+Our goal is to implement an external document management service to hold and provide uploaded documents. Ideally the document management service would be independent of caluma.
```
Comment (winged) [inline docs/original_alexandria_rfc.md:5]: I think the "goal" section here is a good fragment to copy to the README to give a short introduction

File: docs/original_alexandria_rfc.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+## Rationale
```
Comment (winged) [inline docs/original_alexandria_rfc.md:1]: Why start with H2 headings? `s/^##/#/g` :-)

File: docs/original_alexandria_rfc.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+Documents can currently only be added to a form via file question. These do not provide search, tags, thumbnails and some other features needed for a document management service. File questions can only be used inside a form.
```
Comment (winged) [inline docs/original_alexandria_rfc.md:2]: I'd rephrase this slightly to reference caluma. Might not be obvious to "outsiders".

```suggestion
Documents in Caluma can currently only be added to a form via file question. These do not provide search, tags, thumbnails and some other features needed for a document management service. File questions can only be used inside a form.
```

**Comment by open-dynaMIX** [comment]
@winged Thanks for the review! Shouldn't we better consolidate the RFC in the Caluma issue and then copy/paste it here (like we did with emeis)?

BTW: The README is still on cookiecutter level. I will work on that as soon as the configuration part is finished in alexandria.
